src/core/dash_internal.h

```diff
@@ -1587,6 +1588,13 @@ auto Segment<Key, Value, Policy>::BumpUp(uint8_t bid, SlotId slot, Hash_t key_ha
   constexpr unsigned kLastSlot = kNumSlots - 1;
   assert(swapb.GetBusy() & (1 << kLastSlot));
 
+  // Don't move sticky items back to the stash because they're not evictable
+  // TODO: search for first swappable item
+  if (!bp.CanBumpDown(swapb.key[kLastSlot])) {
+    target.SetStashPtr(stash_pos, fp_hash, &next);
```

Comment:
I guess thats all that needed. Our item still "belongs" to the same base bucket, same slot, same everything.

Reply:
yes.